### ISSUE TITLE
Make tests for do_item_item_action

### ISSUE THREAD

**Issue by SnowballAntrobus** [uchicago-cs/chiventure#182]
The goal is to make multiple tests for do_item_item_action. They should be comprehensive, covering all possible outcomes. These tests will compare the printed string output of the function with a given correct string.

This was a branch off issue #177

**Comment by SnowballAntrobus** [comment]
Currently the makefile does not have a rule to make action_management.a. This is easily solved by just running make at the top level directory. I believe the new rule will be added in #150, but if not this is a reminder to do it.

**Comment by SnowballAntrobus** [comment]
Currently the tests will not make as we are waiting on Game State's pull request to master.

**Comment by SnowballAntrobus** [comment]
Created task #188 to remember to add new tests after function has been updated to use game state functions.

**Comment by SnowballAntrobus** [comment]
Work for this issue is on branch: actions/itemitemtests

**Comment by SnowballAntrobus** [comment]
Closing statement:
We wrote basic tests for the function do_item_item_action. These tests focused on making sure that the correct string is being printed when correct and incorrect action kinds are used and when allowed and not allowed actions are used. In future work we would make testing simpler by making the return value of the function an error code int. Additionally, we will make more tests for this function once it is updated to change game state.
